core/src/main/scala/cats/syntax/reducible.scala

```diff
@@ -29,5 +29,9 @@ final class ReducibleOps0[F[_], A](private val fa: F[A]) extends AnyVal {
    * a: String = "foo321"
    * }}}
    * */
-  def reduceMapK[G[_], B](f: A => G[B])(implicit F: Reducible[F], G: SemigroupK[G]): G[B] = F.reduceMapK[G, A, B](fa)(f)
+  def reduceMapK[G[_], B](f: A => G[B])(implicit F: Reducible[F], G: SemigroupK[G]): G[B] =
+    F.reduceLeftTo(fa)(f)((b, a) => G.combineK(b, f(a)))
```

Comment:
Why did you change this? In general we want to avoid having implementation logic in syntax type classes.

Reply:
Yes, my bad, I'll remove this diff.